app/src/models/stash-entry.ts

```diff
@@ -14,6 +14,17 @@ export interface IStashEntry {
   readonly files: StashedFileChanges
 }
 
+export interface IStash {
```

Comment:
This actually does return a single stash located at `.git/logs/refs`. It's split by _type_ however. I did not name it `IStashEntries because this object contains two collections that are stash entries and renaming it do `IStashCollection` feels redundant since a stash is a collection.